package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-bootstrap-controls",
-  "version": "0.0.8",
+  "version": "0.0.9",
```

Comment:
We should handle version bumping after a PR has been accepted and merged to master.